src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -176,7 +176,7 @@ public function testSilentHelp()
     }
 
     /**
-     * @expectedException        \InvalidArgumentException
+     * @expectedException        Symfony\Component\Console\Exception\CommandNotDefinedException
```

Comment:
Can you fix this? We are using a leading `\` everywhere else in Symfony for such PHPUnit annotations.